pkg/plugins/compiler.go

```diff
@@ -51,6 +53,13 @@ func DefaultSrcRoot() (string, error) {
 	}
 	nope = append(nope, root)
 
+	_, filename, _, _ := runtime.Caller(1)
```

Comment:
What is this?  Add a comment.

Reply:
This is to get the current file path so that we can find the path to `kusotmize` directory when $GOPATH is not set. This is needed in the testing.
I'll add a comment.